### ISSUE TITLE
Describe how to use the image with podman

### ISSUE THREAD

**Issue by przem123** [IBM/ibm-spectrum-scale-csi-operator#77]
You dont describe how to use the image with podman.

**Comment by przem123** [comment]
Any update on this? On Openshift 4.2 I have podman, not docker.

**Comment by mew2057** [comment]
If you're using Openshift 4.2, you _should_ be able to just use the `oc apply -f` commands. The  README calls out some `kubectl` commands, but they can easily be swapped for `oc` and since the images live in the quay.io repo podman should be served  those images by Openshift.


See https://github.com/IBM/ibm-spectrum-scale-csi-operator#option-a-manually
---

It also looks like podman uses the same commands as docker:

https://developers.redhat.com/blog/2019/02/21/podman-and-buildah-for-docker-users/

---

I currently don't have a podman environment to  test this, @vhu or @yadaven, do your environments use podman?

**Comment by przem123** [comment]
Yes, kubectl commands can be used with oc, but the problem is that the image is stored in docker registry only. You require docker for the command `operator-sdk build csi-scale-operator` I do not want to install and use it on my worker nodes at all.

Docker engine is not supported on Openshift 4. Please provide a way to use the operator with podman only.

**Comment by mew2057** [comment]
Operator SDK has `operator-sdk build --image-builder podman` for podman builds

---

@vhu we may need to have a podman registry.
